spec/lib/rmt/cli/import_spec.rb

```diff
@@ -117,14 +124,20 @@
 
 
     context 'with existing lockfile' do
-      before { allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError) }
+      before do
+        allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError)
+      end
 
       it 'handles lockfile exception' do
         FakeFS.with_fresh do
           FileUtils.mkdir_p path
           File.write("#{path}/repos.json", repo_settings.to_json)
 
-          expect { command }.to output(/Process is locked/).to_stdout.and output('').to_stderr
+          expect(described_class).to receive(:exit)
+          expect(File).to receive(:read).with(RMT::Lockfile::LOCKFILE_LOCATION).and_return(pid)
```

Comment:
You have a lot of duplicated or very similar lines in these new specs. Please try to join them in an shared_example, if possible.